.github/workflows/ci.yml

```diff
@@ -0,0 +1,196 @@
+name: Tests 1.x
+
+on:
+  push:
+    branches:
+    - 1.x
+  pull_request:
+    branches:
+    - 1.x
+
+jobs:
+  smoke:
```

Comment:
The idea of making the CI work like this is to catch silly errors quickly without having to start 10 containers at once.

If we add a linting step (check code follows a certain format), that would go before theis smoke test so that linting errors are firtst checked and nothing runs if they do not pass.

Reply:
But this is running the same tests as Linux. So do we really need this?